src/main/resources/templates/dev-hub/authentication.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en-GB" layout:decorate="~{layouts/baseLayout}"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="http://www.thymeleaf.org">
+
+<head>
+<title th:utext="${templateTitle}"></title>
+</head>
+
+<body>
+
+    <div layout:fragment="content">
+        <h1 class="govuk-heading-xl">Authentication</h1>
+        <h2 class="govuk-heading-l">API key authentication</h2>
+
+        <p class="govuk-body">The Companies House API requires
+            authentication credentials, in the form of an API key, to be
+            sent with each request.</p>
+
+        <p class="govuk-body">
+            To obtain an API key, go to <a href=https://www.example.com />Your
+            applications</a> and register the application with the Companies
+            House Developer Hub as an
+            <code>API Key</code>
+            application. This will allocate a unique key to the
+            application which can be sent with any
+            <code>GET</code>
+            request for a public resource served by the Companies House
+            API.
+        </p>
+
+        <h3 class="govuk-heading-m">Sending the API key</h3>
+        <p class="govuk-body">
+            The Companies House API uses <a
+                href="https://www.example.com/">HTTP Basic
```

Comment:
Same comment about the gov uk link style

Reply:
Done